rollup.config.js

```diff
@@ -4,7 +4,7 @@ import external from "rollup-plugin-peer-deps-external";
 import postcss from "rollup-plugin-postcss";
 import resolve from "rollup-plugin-node-resolve";
 import url from "rollup-plugin-url";
-import svgr from "@svgr/rollup";
+// import svgr from "@svgr/rollup";
```

Comment:
@fernandoabolafio Could please confirm it solves the problem for you locally as well ?

Reply:
@fernandoabolafio  
Upgrading to the latest also solving the error, so if the library is used we could keep using it :tada:  :tada: :tada: